WireAPI/Sources/WireAPI/APIs/VerificationCodeAPI/VerificationCodeAPIV8.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class VerificationCodeAPIV8: VerificationCodeAPIV7 {
+
+    override var apiVersion: APIVersion {
+        .v8
+    }
+
+    override func requestVerificationCode(for email: String) async throws {
+
+        let path = "\(pathPrefix)/verification-code/send"
+
+        let body = try JSONEncoder.defaultEncoder.encode(
+            RequestVerificationCodeRequestBodyV8(
+                action: "login",
+                email: email
+            )
+        )
+
+        let request = try URLRequestBuilder(path: path)
+            .withBody(body, contentType: .json)
+            .build()
+
+        let (data, response) = try await apiService.executeRequest(
+            request,
+            requiringAccessToken: false
```

Comment:
suggestion: since this is not an authenticated request, you could depend on `NetworkService` instead of `APIService`, which would simplify the dependency graph.

Reply:
I briefly had a look, but I think we should do this in another PR.